### PR TITLE
[Manta] `v0.9.12` runtime weights for Calamari runtime

### LINKED ISSUE #235: Upgrade dependencies to v0.9.12-1
There's a new polkadot release v0.9.12-1
[5feed981cf0fe671447eabaa9c0d235a8dd0003e](https://github.com/paritytech/polkadot/releases/tag/v0.9.12-1)

And related deps:
substrate -> [d76f39995315ec36980908e4b99709bd14927044](https://github.com/paritytech/substrate/tree/polkadot-v0.9.12)
cumulus -> [935bac869a72baef17e46d2ae1abc8c0c650cef5](https://github.com/paritytech/cumulus/tree/polkadot-v0.9.12)

### PR DESCRIPTION
**Pull request by ghzlatarev**
## Description

closes: #235

* New weight files for Calamari runtime. Generated on `ghzlatarev/polkadot-v0.9.12` https://github.com/Manta-Network/Manta/actions/runs/1426704200
* CI name changes after default git branch change
* Updated the weight file generation output in order to fix a compilation bug.
* Add `pallet_multisig` to weights generation workflow
---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (`manta` or `manta-pc`) with right title (start with [Manta] or [Manta-PC]),
- [x] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [ ] Wrote unit tests.
- [x] Updated relevant documentation in the code.
- [x] Re-reviewed `Files changed` in the Github PR explorer.
- [x] If runtime changes, need to update the version numbers properly:
   * `authoring_version`: The version of the authorship interface. An authoring node will not attempt to author blocks unless this is equal to its native runtime.
   * `spec_version`: The version of the runtime specification. A full node will not attempt to use its native runtime in substitute for the on-chain Wasm runtime unless all of spec_name, spec_version, and authoring_version are the same between Wasm and native.
   * `impl_version`: The version of the implementation of the specification. Nodes are free to ignore this; it serves only as an indication that the code is different; as long as the other two versions are the same then while the actual code may be different, it is nonetheless required to do the same thing. Non-consensus-breaking optimizations are about the only changes that could be made which would result in only the impl_version changing.
   * `transaction_version`: The version of the extrinsics interface. This number must be updated in the following circumstances: extrinsic parameters (number, order, or types) have been changed; extrinsics or pallets have been removed; or the pallet order in the construct_runtime! macro or extrinsic order in a pallet has been changed. If this number is updated, then the spec_version must also be updated 
- [x] If needed, notify the committer this is a draft-release and a tag is needed after merging the PR.
- [x] Verify benchmarks & weights have been updated for any modified runtime logics

### CODE REVIEW

File: runtime/calamari/src/lib.rs [hunk 1]
```diff
@@ -222,7 +222,7 @@ impl pallet_balances::Config for Runtime {
 	type Event = Event;
 	type ExistentialDeposit = NativeTokenExistentialDeposit;
 	type AccountStore = frame_system::Pallet<Runtime>;
-	type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
+	type WeightInfo = weights::pallet_balances::SubstrateWeight<Runtime>;
```
Comment (ghzlatarev) [inline runtime/calamari/src/lib.rs:225]: @stechu @Dengjianping 
 `SubstrateWeight` is the new old name of struct which implements the `WeightInfo` trait. That's what our new template file outputs.

File: .github/resources/frame-weight-template.hbs [hunk 2]
```diff
@@ -48,7 +48,7 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
 {{~#if (eq pallet "frame_system")}}
 impl<T: crate::Config> WeightInfo for SubstrateWeight<T> {
 {{~else}}
-impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
+impl<T: frame_system::Config> {{pallet}}::WeightInfo for SubstrateWeight<T> {
```
Comment (ghzlatarev) [inline .github/resources/frame-weight-template.hbs:51]: Had to specify which pallet's WeightInfo trait explicitly in order to compile.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -18,7 +18,6 @@ Before we can merge this PR, please make sure that all the following items have
 checked off. If any of the checklist items are not applicable, please leave them but
 write a little note why.
 
-- [ ] Targeted PR against correct branch (`manta` or `manta-pc`) with right title (start with [Manta] or [Manta-PC]),
```
Comment (ghzlatarev) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: @stechu @Dengjianping 

We only have the `manta` branch so this doesn't make sense.
Maybe we can classify by runtime ?

Comment (stechu) [reply .github/PULL_REQUEST_TEMPLATE.md:21]: we do have `dolphin` branch for testnet.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ Before we can merge this PR, please make sure that all the following items have
...
+- [ ] Targeted PR against correct branch (`manta` or `dolphin`) with right title (start with [Manta] or [Dolphin]),
```
Comment (Dengjianping) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: Can we add one more tag `Calamari`?

Comment (stechu) [reply .github/PULL_REQUEST_TEMPLATE.md:21]: We don't have Calamari branch.

**Review by stechu — APPROVED** [review]

**Comment by ghzlatarev** [comment]
Reopening this with signed commits.

**Pull request closed without merging**